sdk/ml/azure-ai-ml/swagger/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-10-01-preview/registries.json

```diff
@@ -111,6 +111,7 @@
           "RegistryManagement"
         ],
         "summary": "Delete registry.",
+        "x-ms-long-running-operation": true,
```

Comment:
Could you validate if the status 202 should read like this , because right now I don't see retry, timeout etc... 

         "202": {
            "description": "Accepted",
            "headers": {
              "x-ms-async-operation-timeout": {
                "description": "Timeout for the client to use when polling the asynchronous operation.",
                "type": "string",
                "format": "duration"
              },
              "Location": {
                "description": "URI to poll for asynchronous operation result.",
                "type": "string"
              },
              "Retry-After": {
                "description": "Duration the client should wait between requests, in seconds.",
                "type": "integer",
                "format": "int32",
                "maximum": 600,
                "minimum": 10
              }
            }
          }
        },

Reply:
I changed the swagger to resemble this.